app/services/data_exports/aggregate_metadata_file_generator.rb

```diff
@@ -7,28 +7,28 @@ class AggregateMetadataFileGenerator
     class << self
       # Public: add metadata csv file to group folder
       def generate_group_file(transcriptions, output_folder)
-        @metadata_file = File.join(output_folder, 'transcriptions_metadata.csv')
+        metadata_file = File.join(output_folder, 'transcriptions_metadata.csv')
```

Comment:
Not blocking, you could dry this up a little by pushing this setup file name into the `generate_csv` method and pass in the `output_folder` and other param. It could call another private method or inline the setup file name.

Reply:
Good call.